### PR TITLE
Adding everyone monitor

### PR DESCRIPTION
**Pull request by YorkAARGH**
This monitor will give whom ever tags a fake `everyone` role, the same everyone role, thus annoying a lot of people. _kappa_

### CODE REVIEW

File: monitors/everyone.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  async run(msg) {
+    if (!msg.member || !msg.guild.configs.everyoneRole || !msg.mentions.roles.size || !await msg.hasAtLeastPermissionLevel(1)) return;
+    const everyone = msg.guild.roles.get(msg.guild.configs.everyoneRole);
+    if (!everyone) {
+      await msg.guild.configs.reset("everyoneRole");
+      return;
+    }
+    if (!msg.mentions.roles.has(everyone.id)) return;
+    await msg.member.roles.add(everyone);
```
Comment (MrJacz) [inline monitors/everyone.js]: Should check if the bot has manage_roles perms and the everyone role position isn't higher then the author's highest role position

File: monitors/everyone.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		super(...args, { ignoreOthers: false });
+	}
+
+	async run(msg) {
+		if (!msg.member ||
+			!msg.guild.configs.everyoneRole ||
+			!msg.mentions.roles.size ||
+			msg.guild.me.roles.highest.position <= msg.member.roles.highest.position ||
+			!msg.guild.me.permissions.has(FLAGS.MANAGE_ROLES) ||
+			!await msg.hasAtLeastPermissionLevel(1)
```
Comment (Pandraghon) [inline monitors/everyone.js]: Is there any particular reason not to use `botPerms` nor `permLevel`?

Comment (kyranet) [reply monitors/everyone.js]: Because it's a monitor.

Comment (tech6hutch) [reply monitors/everyone.js]: This could maybe be made cleaner by moving it to a shouldRun method, since it's determining whether it should run

**Pull request merged**